fastddsspy_tool/test/application/test.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Still missing a README file that explains this test architecture and slightly how to add a new test case.
Explain: 
- what is test.py, 
- what is test_class.py and more or less what it does (variables and its routines/workflows)
- how a new test_case file is added and some things that may take care of: if the name means something (I think no longer does), how to change the valid_output function, etc.